alpha/lib/data/live/kLiveEntryRecordingOptions.php

```diff
@@ -23,10 +23,15 @@ public function setShouldCopyEntitlement($shouldCopyEntitlement)
 	}
 	
 	/**
+	 * @var int
 	 * @return boolean
 	 */
-	public function getShouldCopyEntitlement()
+	public function getShouldCopyEntitlement($partnerId)
 	{
+		if ( PermissionPeer::isValidForPartner(PermissionName::FEATURE_LIVE_STREAM_COPY_ENTITELMENTS, $partnerId) )

```

Comment:
This way you prevent disabling it using the API.
I suggest that you will ovveride KalturaEntryService::prepareEntryForInsert in KalturaLiveEntryService and check if this attribute is null, and only in such case, use the partner defaults.